app/src/main/java/com/jetbrains/handson/mpp/mobile/AdvancedSearchActivity.kt

```diff
@@ -13,10 +13,12 @@ import android.widget.TimePicker
 import kotlinx.android.synthetic.main.activity_pop_up_window.*
 import java.util.*
 
-class AdvancedSearchActivity : AppCompatActivity(), DatePickerDialog.OnDateSetListener,
+class AdvancedSearchActivity : AppCompatActivity(), AdvancedSearchContract.View, DatePickerDialog.OnDateSetListener,
         TimePickerDialog.OnTimeSetListener {
+
+        private lateinit var presenter: AdvancedSearchPresenter
+
         lateinit var textView: TextView
-        lateinit var button: Button
         var day = 0
```

Comment:
I'm slightly confused by this combination of <unitOfTime> variables and my<unitOfTime> variables, what's your intention here?

Reply:
I think this is just leftover from when I was figuring out how to use the date/time picker - have removed the my_ variables

Reply:
Ah okay, makes more sense now, thanks :)